packages/explorer/src/EmbeddedExplorer.ts

```diff
@@ -153,11 +161,12 @@ export class EmbeddedExplorer {
     iframeElement.src = this.embeddedExplorerURL;
 
     iframeElement.id = IFRAME_DOM_ID(this.uniqueEmbedInstanceId);
-    iframeElement.setAttribute(
-      'style',
-      'height: 100%; width: 100%; border: none;'
-    );
-
+    if (this.options.allowDynamicStyles !== false) {
```

Comment:
Adding a comment so I don't come around and "simplify" this to just `if (this.options.allowDynamicStyles) {...}`
```suggestion
    // default to `true` (`true` and `undefined` both ok)
    if (this.options.allowDynamicStyles !== false) {
```